packages/cli/src/commands/setup/auth/templates/dbAuth.function.ts.template

```diff
@@ -140,6 +140,21 @@ export const handler = async (event, context) => {
       resetTokenExpiresAt: 'resetTokenExpiresAt',
     },
 
+    // Specifies attributes on the cookie that dbAuth sets in order to remember
+    // who is logged in. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#restrict_access_to_cookies
+    cookie: {
```

Comment:
@cannikin can you confirm which of the keys in these settings are required and which are optional please?